src/client.ts

```diff
@@ -38,15 +58,25 @@ export async function* fetchJsonPaginated<T>(
   }
 }
 
-export function fetchPointerChanges(
+export async function* fetchPointerChanges(
```

Comment:
is this a breaking change of the lib? If that's the case remember to upgrade the major version when releasing

Reply:
Yes, will be major version